### ISSUE TITLE
Investigate px/rem sizing units

### ISSUE THREAD

**Issue by retrixe** [retrixe/concinnity#38]
We should have a `Spacer` component in Heliodor, `<br>` carries semantic meanings that disrupt usability and accessibility

**Comment by retrixe** [comment]
Also, https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css

We should have consistent units
